ui/pages/confirm-add-suggested-token/confirm-add-suggested-token.js

```diff
@@ -117,11 +106,40 @@ const ConfirmAddSuggestedToken = (props) => {
     );
   }, [suggestedAssets, tokens, t]);
 
+  const handleAddTokensClick = useCallback(async () => {
+    await Promise.all(
+      suggestedAssets.map(async ({ asset, id }) => {
+        await dispatch(acceptWatchAsset(id));
+
+        metricsEvent({
```

Comment:
[REQ] change this to `trackEvent` which is the name of the context function for the new context, but also the useNewMetricsEvent hook calls it metricsEvent which is wrong so i get why there is like 5 layers of misdirection. 

Maybe i was trying to encode job security here? 🤷🏻

Reply:
updated, thank you! @brad-decker